sdk/core/System.ClientModel/tests/gen/System.ClientModel.SourceGeneration.Tests.csproj

```diff
@@ -45,12 +45,13 @@
     <Compile Include="..\ModelReaderWriterTests\MrwTestBase.cs" LinkBase="SharedTests" />
     <Compile Include="..\ModelReaderWriterTests\RoundTripStrategy.cs" LinkBase="SharedTests" />
   </ItemGroup>
-  
+
   <ItemGroup>
     <Compile Include="..\client\ModelReaderWriter\Models\DiscriminatorSet\BaseModel.cs" LinkBase="Shared\Models" />
     <Compile Include="..\client\ModelReaderWriter\Models\DiscriminatorSet\ModelX.cs" LinkBase="Shared\Models" />
     <Compile Include="..\client\ModelReaderWriter\Models\DiscriminatorSet\ModelY.cs" LinkBase="Shared\Models" />
     <Compile Include="..\client\ModelReaderWriter\Models\DiscriminatorSet\UnknownBaseModel.cs" LinkBase="Shared\Models" />
+    <Compile Include="..\client\ModelReaderWriter\Models\ExperimentalModel.cs" LinkBase="Shared\Models" />
```

Comment:
I think we were trying to simulate pulling this in via a dependency?  Adding it to shared source negates that I believe.